packages/kubernetes-1.20/kubelet-config

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
We'll need to make sure we get the changes from:
* https://github.com/bottlerocket-os/bottlerocket/pull/1388
* https://github.com/bottlerocket-os/bottlerocket/pull/1465
* https://github.com/bottlerocket-os/bottlerocket/pull/1472

Reply:
And #1408 changes variant Cargo.toml and package Cargo.toml patterns a bit.